src/snapshots/ruff__linter__tests__B015_B015.py.snap

```diff
@@ -0,0 +1,37 @@
+---
+source: src/linter.rs
+expression: checks
+---
+- kind: UselessComparison
+  location:
+    row: 3
+    column: 2
```

Comment:
Weird. I think the column should be 0.

Reply:
I think this is the location of the `==`?

Reply:
Yes.